### ISSUE TITLE
Different types of cargo containers

### ISSUE THREAD

**Issue by sqrw** [sqrw/sqrw.github.io#19]
I think that exists, need to verify in PU.

**Comment by Jehette** [comment]
Yes, there are the small hand held box currently and then they have the 1 SCU sized ones, which also changes depending on which type of material you are transporting.

**Comment by sqrw** [comment]
Lets add a screenshot and mark it as in alpha
